test/metabase/notification/payload/impl/card_test.clj

```diff
@@ -612,3 +612,28 @@
                                           "ngoc@metabase.com"]
                              :filters    nil}}
                  (mt/latest-audit-log-entry :alert-send (:id notification)))))))))
+
+(deftest progress-value-setting-test
```

Comment:
Let's mark all the tests in both test files as `^:parallel` and they could be split up so each `(testing...` is in a separate deftest. That will make the tests run better in parallel and make debugging easier.